app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
   default from: "droply.supplies@gmail.com"
-  layout "mailer"
+  layout 'mailer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.